### DISCUSSION TITLE
笔电连手机热点时无法找到手机

### DISCUSSION THREAD

**Discussion by xsgteamx** [localsend/localsend#1146] [L-简体中文]
当手机开热点共享网络的场景下，笔电链接 手机热点 时
笔电无法在 “发送” 界面找到手机
但手机可找到笔电 且可发送文件等信息
个人排查的问题 可能的原因：此时手机的热点产生的虚拟IP 例如为：`192.168.60.191` 作为了笔电 的网关
可能产生了某种不可知的bug?
![Snipaste_2024-02-19_15-25-25](https://github.com/localsend/localsend/assets/31960432/e1c0f51b-d7c9-461f-b14e-043e72535753)
笔电可能影响网络的接口 有使用VMware 虚拟网卡 固禁用了虚拟网卡排除了 可能会产生干扰的因素之后 再试
均失败
![电脑网口配置信息](https://github.com/localsend/localsend/assets/31960432/43cfd589-42d7-4ed7-82a3-089205940801)
![手机软件界面](https://github.com/localsend/localsend/assets/31960432/34d192a3-9c77-4215-83a0-c68c27c09085)

而通过笔电的 "分享为链接" 让手机访问时 发现PC可识别到手机
请问各位 有知道这是怎么回事吗?

**Comment by Newdea** [comment]
遇到同样的问题，而且我的手机也找不到电脑，两个相互都找不到，要怎么办

**Comment by dorannn** [comment]
设置为连接分享 然后修改连接的ip进行访问试试
广播的网段应该是在192.168网段，但是实际可访问的是0.0.0.0

**Reply by Newdea** [reply]
> 设置为连接分享 然后修改连接的ip进行访问试试

多谢答复，没明白这个什么意思，不是直接连手机的热点就行么，电脑还需要再设置？我看我电脑是`自动(DHCP)`,  IP就是你说的192.168网段吧。但还是彼此找不到。
![image](https://github.com/user-attachments/assets/f7292c9d-cd92-495c-87f3-29f68d2fbf1d)

**Reply by dorannn** [reply]
![image](https://github.com/user-attachments/assets/69edc2b2-b0ff-400a-86f1-f26c40922f69)

![image](https://github.com/user-attachments/assets/bbb8259a-c7a8-42c0-9063-186b880a9322)
你看一下电脑端的localsend 这个界面里网段是否包含你上述的ip
如果没有 下载最新版试试 我记得原来是没有的，后来就有了

**Reply by Newdea** [reply]
多谢回复。
是release页面上的最新版的，但也没有，三个IP ...89/35/123都试了，全是错误。我看手机上的标签是`#15，#35`， 电脑上是`#1，#89`
![image](https://github.com/user-attachments/assets/8b420ca3-cf29-43f3-9b6a-fc357414f8bd)

**Reply by dorannn** [reply]
不清楚了 我刚才试了一下手机和平板是可以热点发现到的
